docs/scheme-api/.gitignore

```diff
@@ -2,3 +2,4 @@
 lepton-scheme.info
 stamp-vti
 version.texi
+lepton-scheme.html/*
```

Comment:
This is superfluous here. (BTW, what command should be issued to make the directory, `make html`?). Please put it into a separate commit with appropriate description.
Anything other suits me well, thank you!

Reply:
Ah, OK, just checked it.

Reply:
I use these commands in my install script:

`cd docs/scheme-api/`
`make html`
`make install-html`